bindings/python/CONTRIBUTING.md

```diff
@@ -0,0 +1,388 @@
+# Contributing to PyMongoArrow
+
+Contributions to PyMongoArrow are always encouraged. Contributions can be as simple
+as minor tweaks to the documentation. Please read these guidelines
+before sending a pull request.
+
+## Bugfixes and New Features
+
+Before starting to write code, look for existing
+[tickets](https://jira.mongodb.org/browse/ARROW) or [create
+one](https://jira.mongodb.org/browse/ARROW) for your specific issue or
+feature request. That way you avoid working on something that might not
+be of interest or that has already been addressed.
+
+## Supported Interpreters
+
+PyMongoArrow supports CPython 3.7+ and PyPy3.8+. Language features not
+supported by all interpreters can not be used.
+
+## Style Guide
+
+PyMongo follows [PEP8](http://www.python.org/dev/peps/pep-0008/)
```

Comment:
-PyMongo follows [PEP8](http://www.python.org/dev/peps/pep-0008/)
-including 4 space indents and 79 character line limits.
+PyMongoArrow uses [Ruff](https://docs.astral.sh/ruff/) for formatting and linting,
+with a line length of 100.

Reply:
Fixed